enginetest/queries/insert_queries.go

```diff
@@ -846,15 +846,6 @@ var InsertScripts = []ScriptTest{
 ) insert into xy (x,y) select x+5,y+5 from a;`,
 				Expected: []sql.Row{{sql.OkResult{RowsAffected: 2, InsertID: 0}}},
 			},
-			{
-				Query: `With a as (
-  With b as (
-    Select sum(x) as x, y from xy where x < 2 group by y
-  )
-  Select * from b d
-) insert into xy (x,y) select x+9,y+9 from a;`,
-				Expected: []sql.Row{{sql.OkResult{RowsAffected: 2, InsertID: 0}}},
-			},
```

Comment:
Is it worth keeping this test query? Or is this one still busted?

Reply:
this one is simpler, i'll swap them, we just need a test with Insert.Source->...TableAlias->Subquery